### PR TITLE
Debug magma on crusher

### PR DESCRIPTION
**Pull request by mewall**
Modifications to support cmake package bug fix for BML on OLCF crusher

- - -
This change is [Reviewable](https://reviewable.io/reviews/lanl/qmd-progress/244)

### CODE REVIEW

**Review by cnegre — APPROVED** [review]

File: build.sh [hunk 1]
```diff
@@ -35,7 +35,6 @@ EOF
     echo "CC                 Path to C compiler       (default is ${CC})"
     echo "CXX                Path to C++ compiler     (default is ${CXX})"
     echo "FC                 Path to Fortran compiler (default is ${FC})"
-    echo "BML_ROCSPARSE      Build with rocSPARSE     (default is ${BML_ROCSPARSE})"
```
Comment (nicolasbock) [inline build.sh:38]: Why do you remove this line? The variable `BML_ROCSPARSE` is still being used.

**Review by nicolasbock — CHANGES_REQUESTED** [review]

**Comment by mewall** [comment]
Thanks for catching that. It's not needed any more with the BML package revision. I changed CMakelists.txt and removed other refs to BML_ROCSPARSE in build.sh.

________________________________
From: Nicolas Bock ***@***.***>
Sent: Monday, February 6, 2023 8:40:46 PM
To: lanl/qmd-progress
Cc: Wall, Michael E; Author
Subject: [EXTERNAL] Re: [lanl/qmd-progress] Debug magma on crusher (PR #244)


@nicolasbock requested changes on this pull request.

________________________________

In build.sh<https://urldefense.com/v3/__https://github.com/lanl/qmd-progress/pull/244*discussion_r1098141880__;Iw!!Bt8fGhp8LhKGRg!GreLoMHHsVkwIcvpco7_fIFJAgJBpHYRw2ODkVLiLLTFT834Owq6V8GhroEyQXznYG2Uf5pfWOoZUaRrPOo4jWrj$>:

> @@ -35,7 +35,6 @@ EOF
     echo "CC                 Path to C compiler       (default is ${CC})"
     echo "CXX                Path to C++ compiler     (default is ${CXX})"
     echo "FC                 Path to Fortran compiler (default is ${FC})"
-    echo "BML_ROCSPARSE      Build with rocSPARSE     (default is ${BML_ROCSPARSE})"


Why do you remove this line? The variable BML_ROCSPARSE is still being used.

**Review by jeanlucf22 — CHANGES_REQUESTED** [review]
What do you mean? Are we dropping off support for rocsparse?

**Comment by mewall** [comment]
The BML package now handles the rocSPARSE dependency. Please see the note on the commit with reference to the BML PR#...

________________________________
From: Jean-Luc Fattebert ***@***.***>
Sent: Tuesday, February 7, 2023 8:19:04 AM
To: lanl/qmd-progress
Cc: Wall, Michael E; Author
Subject: [EXTERNAL] Re: [lanl/qmd-progress] Debug magma on crusher (PR #244)


@jeanlucf22 requested changes on this pull request.

What do you mean? Are we dropping off support for rocsparse?

**Comment by mewall** [comment]
See


https://github.com/lanl/bml/blob/master/src/BMLConfig.cmakein

________________________________

**Comment by jeanlucf22** [comment]
@mewall sorry, I thought I was looking at BML. I agree, rocsparse should not be anywehere in the build scripts here

**Review by nicolasbock — CHANGES_REQUESTED** [review]
Thanks for the clarification!

Can we remove

https://github.com/lanl/qmd-progress/blob/48c9a7c210c4a4e3028453118bf95aff166e8f1f/scripts/build_crusher_rocsparse_cce.sh#L16

also then?

**Comment by mewall** [comment]
Done

________________________________
From: Nicolas Bock ***@***.***>
Sent: Tuesday, February 7, 2023 9:56:44 AM
To: lanl/qmd-progress
Cc: Wall, Michael E; Mention
Subject: [EXTERNAL] Re: [lanl/qmd-progress] Debug magma on crusher (PR #244)


@nicolasbock requested changes on this pull request.

Thanks for the clarification!

Can we remove

https://github.com/lanl/qmd-progress/blob/48c9a7c210c4a4e3028453118bf95aff166e8f1f/scripts/build_crusher_rocsparse_cce.sh#L16<https://urldefense.com/v3/__https://github.com/lanl/qmd-progress/blob/48c9a7c210c4a4e3028453118bf95aff166e8f1f/scripts/build_crusher_rocsparse_cce.sh*L16__;Iw!!Bt8fGhp8LhKGRg!Fl62TMZF5cxdIe2NvuQPjXhVMKr5P0GpGfL0vOny-VWqv3oymZogcRusMjZjWiD7fQwb3BTSiOCY2Re27erd8wYr$>

also then?

**Review by nicolasbock — APPROVED** [review]
Thanks!

**Pull request merged**